.github/workflows/replace-docs.yml

```diff
@@ -33,6 +33,7 @@ jobs:
       run: |
         rm -rf ./site/${{ env.VERSION }}
         rm -rf ./site/.git/
+        ls -la
```

Comment:
This just a sanity check? Do we need this?

Reply:
yes, but this workflow is only a "backup" workflow that can be triggered manually if the "publish" workflow fails, which for odd reasons sometimes does. 

If you don't mind, I'm going to leave it in as it's useful in case the VERSION gets set incorrectly or other odd folders like the .git show up that are not expected

Reply:
Works for me!